routemaster/webhooks.py

```diff
@@ -16,7 +16,7 @@ class WebhookResult(enum.Enum):
     FAIL = 'fail'
 
 
-WebhookRunner = Callable[[str, str, bytes], WebhookResult]
+WebhookRunner = Callable[[str, str, bytes, str], WebhookResult]
```

Comment:
We should probably `NewType` these. With four arguments this is beginning to get unwieldy.

Reply:
Unfortunately it doesn't look like you can use `NewType` like this, or I'm reading the docs wrong and using it incorrectly.